deploy/apecloud-mysql/config/mysql8-config.tpl

```diff
@@ -94,15 +94,13 @@ mysqlx-port=33060
 mysqlx=0
 
 datadir={{ $data_root }}/data
+log_error=/data/mysql/log/mysqld-error.log
```

Comment:
Is this a necessary change?

Reply:
No, will move this change to another PR.